app/views/posts/show.html.erb

```diff
@@ -1,22 +1,37 @@
-
- <div class="container">  
+<div class="container">  
   <div class="post-container">
     <div class="user-details">
       <h2 class="user-name">Post #<%= @post_number %> by <%= @user.name %></h2>
       <p class="user-post-count">Number of comments: <%= @post.comments.count %>,
          Number of likes: <%= @post.likes.count %>
       </p>
     </div>
+    
     <h4><%= @post.title %> by <%= @post.author.name %></h4>
+
+    <div class="like">
+      <%= form_with model: @like, url: user_post_likes_path(user_id: @post.author.id, post_id: @post.id), local: true do |form| %>
+        <%= form.submit "Like", class: "like" %>
+      <% end %>
+    </div>
+
     <p><%= @post.text %></p>
     <div class="comments-container">
       <h4>Comments:</h4>
+      <div class="comment-form">
+        <%= form_with model: [@user, @post, @comment], url: user_post_comments_path(@user, @post), class: "form", local: true do |form| %>
+          <div>
+            <%= form.text_area :text, class: "text-area", placeholder: "Add a comment..." %>
+          </div>
+          <div >
+            <%= form.submit "Add Comment", class: "button comment-button" %>
+          </div>
+        <% end %>
+      </div>
```

Comment:
Good work adding a form for the user to add a comment :+1: However this form is supposed to be in a the `views/comments/new.html.erb` file.